File: app/src/app/features/clients/models/client.model.ts
import { TrackableModel } from "src/app/core/models/trackable.model";
import { TrackableResource } from "src/app/core/resources/trackable.resource";
import { TemplateModel } from "../../templates/models/template.model";
import { Gender } from "../enums/gender.enum";

export class ClientModel extends TrackableModel{
    fullName: string;
    birthDate: Date;
    phoneNumber: string;
    gender: Gender;
    workPlacesIds: Array<number>;
    isActive: boolean;
    template: TemplateModel
}

File: api/BeHealth.Business/Models/Clients/ClientModel.cs
using BeHealth.Business.Models.DietPlan;
using BeHealth.Presentence.Enums;
using System;
using System.Collections.Generic;
using System.Text;

namespace BeHealth.Business.Models.Clients
{
    public class ClientModel : TrackableModel
    {
        public string FullName { get; set; }

        public DateTime? BirthDate { get; set; }

        public string PhoneNumber { get; set; }

        public GenderEnum? Gender { get; set; }

        public IList<long> workPlacesIds { get; set; }

        public TemplateModel Template { get; set; }
    }
}
